### PR TITLE
Add issue template and pr template

### PR DESCRIPTION
**Pull request by farmdawgnation**
We'll also go ahead and unify the contributing file under the .github
folder.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**[Mailing List](https://groups.google.com/forum/#!forum/liftweb) thread:** REPLACE THIS WITH ML LINK
+
+**Example project link:** LINK TO EXAMPLE PROJECT IF APPLICABLE
```
Comment (Shadowfiend) [inline .github/ISSUE_TEMPLATE.md]: Should we link to the assemble example project blurb here?

Comment (farmdawgnation) [inline .github/ISSUE_TEMPLATE.md]: That seems to still be hosted on Assembla. Should we move it to the repo?

**Comment by farmdawgnation** [comment]
> Should CONTRIBUTING be under .github? It seems like that's a non-Github-specific concept, while the issue template and the PR template are Github-specific.

I could go either way. Nobody seems to read it in any event, but I'll move it back.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Shadowfiend) [inline .github/ISSUE_TEMPLATE.md]: We certainly can yeah, perhaps under `docs/community` or something? Or move it to the Lift site…

**Comment by Shadowfiend** [comment]
> Nobody seems to read it

Absence of evidence is not evidence of absence :)

**Comment by farmdawgnation** [comment]
Hah, fair enough.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (farmdawgnation) [inline .github/ISSUE_TEMPLATE.md]: Yeah. Hm. Let's table that for now. I'll just link to the assembla site. Fixing the diaspora of documentation is a larger problem that the community needs to make some decisions about before we move forward.

**Comment by farmdawgnation** [comment]
Alright, we should be good here.

**Comment by Shadowfiend** [comment]
Lol impatience :p

**Comment by farmdawgnation** [comment]
I needed to open an issue and wanted my template!

**Pull request merged**